changes.md

```diff
@@ -2,6 +2,7 @@
 
 ## Changes in 2.3
 
+* This version works with Moodle 4.0.
```

Comment:
Because version 2.3 has already been released in the plugins database (https://moodle.org/plugins/qtype_oumultiresponse) this new version should be 2.4, which means we should have a new section in this file.